.changeset/nice-files-explode.md

```diff
@@ -0,0 +1,7 @@
+---
+'@remirror/core': minor
```

Comment:
For some reason `changesets` hasn't been picking up changes in the dependencies of  and automatically patching the `remirror` package. I have no idea why. It does it for all other packages. Until we have a fix could you manually be adding the `remirror` package to all dependencies so that it triggers the change.



```suggestion
'@remirror/core': minor
'remirror': patch
```